spark-jobs/src/main/resources/reference.conf

```diff
@@ -20,9 +20,6 @@
 
 # IMPORTANT. Add all sensitive configuration keys to Constants.ConfigKeysToRedact.
 
-# Enceladus version (this is populated automatically during the build).
-enceladus.version=${project.version}
```

Comment:
There are usages of this property leading to messages like 

```
2020-08-31 20:55:41,519 INFO standardization.StandardizationJob$: Enceladus version N/A
```

Look into `utils/src/main/scala/za/co/absa/enceladus/utils/general/ProjectMetadataTools.scala`

Reply:
@Zejnilovic , so your suggestion is to change the implementatoin of `ProjectMetadataTools.getEnceladusVersion`, too?

Reply:
Now, this is the hard question because `ProjectMetadataTools` is in `utils`. Either change:
- the `ProjectMetadataTools`
- or the `CommonJobExecution.obtainSparkSession`'s `enceladusVersion`

I would go with the later. 

Actually have something like `ProjectMetadataTools` in spark-jobs and use the maven plugin there would be probably better. But it is also sed in PerformanceMetricTools

Reply:
I would say it's better to have it only one way, so modify `ProjectMetadataTools`. What do you say about this?

Reply:
There is no problem. Just possibility of breaking something that relies on it.

Reply:
Not sure I follow how (doesn't mean it's not the case).
On the other hand I see, it can be nicely leverage in Menas too.